gui/src/components/LinkButton.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import { Link as ReactStaticLink } from 'react-static'
+
+export default ({ children, to, className }) => (
+  <ReactStaticLink className={className} to={to}>
```

Comment:
A separate PR also has a `LinkButton` so let's come up with better names for both. What makes this a button to begin with and not just a `ReactStaticLink`? I see it predominantly being used by `Button ... component`. Can we just pass in `ReactStaticLink` and remove this wrapper altogether? If not, maybe we can call it `ReactStaticLinkComponent`?

Reply:
👍 for calling this a `ReactStaticLinkComponent`. As an idea, maybe we can have both components in a single `LinkButton.js` as exports with more descriptive names?
@dimroc

Reply:
Maybe in the merge when we conflict ;). For now I would just give it it's own file (rename this to ReactStaticLinkComponent.js) and we can consolidate later if need be.